README.md

```diff
@@ -156,3 +159,66 @@ Set different options for mynetworks option (can be overwrite in postfix-main.cf
   - **empty** => localhost only
   - host => Add docker host (ipv4 only)
   - network => Add all docker containers (ipv4 only)
+
+##### SASL_MECHANISMS
+  - empty => pam
+  - ldap => authenticate against ldap server
+  - shadow => authenticate against local user db
+  - mysql => authenticate against mysql db
+  - rimap => authenticate against imap server
+  - NOTE: can be a list of mechanisms like pam ldap shadow
+
+##### SASL_MECH_OPTIONS
+  - empty => None
+  - e.g. with sasl_mechanism rimap you need to specify the ip-address/servername of the imap server  ==> xxx.xxx.xxx.xxx
+
+##### SASL_LDAP_SERVER
+  - empty => localhost
+
+##### SASL_LDAP_PROTO
+  - empty => ldap://
+  - 1 => ldaps://
+
+##### SASL_LDAP_BIND_DN
+  - empty => anonymous bind
+  - specify an object with priviliges to search the directory tree
+  - e.g. active directory: SASL_BIND_DN=cn=Administrator,cn=Users,dc=mydomain,dc=net
+  - e.g. openldap: SASL_BIND_DN=cn=admin,dc=mydomain,dc=net
+
+##### SASL_LDAP_PASSWORD
+  - empty => anonymous bind
+
+##### SASL_LDAP_SEARCH_BASE
+  - empty => Reverting to SASL_MECHANISM pam
+  - specify the search base
+
+##### SASL_LDAP_FILTER
+  - empty => default filter (uid=%u)
+  - e.g. for active directory: (&(sAMAccountName=%U)(objectClass=person))
+  - e.g. for openldap: (&(uid=%U)(objectClass=person))
+
+##### LDAP
+  - **empty** => LDAP support disabled
+  - 1 => set virtual_mailbox_maps = ldap:/etc/postfix/ldap-users.cf and virtual_alias_maps = ldap:/etc//postfix/ldap-aliases.cf
+  - $SMTP_ONLY must be set to 1
```

Comment:
i implemented this because i needed those features for a kopano (former zarafa) integration. i didn't implement ldap for dovecot, but postfix-ldap and saslauthd-ldap